### DISCUSSION TITLE
Ubuntu Repo still delivering Version 2.0.45 from January 2021

### DISCUSSION THREAD

**Discussion by pflanzenandi** [DNSCrypt/dnscrypt-proxy#2461] [Q&A] [answered]
Is there a way to push Ubuntu to update Dnscrypt-proxy?

**Comment by alpominth** [comment]
I'm sure the only way is to install the latest version manually.

**Reply by pflanzenandi** [reply]
Is there an easy way to update my current Ubuntu APT installation? My current installation is using systemd socket activation.

**Reply by alpominth** [reply, marked as answer]
There is no way because I took a serious Google search for all Debian/Ubuntu packages of the latest versions and I only found old ones.

But don't worry, I found the latest version in ALT Linux repositories: https://packages.altlinux.org/en/sisyphus/binary/dnscrypt-proxy/x86_64/2971982761161710448

All you may do is to convert the RPM package to .deb with [alien](https://www.geeksforgeeks.org/how-to-convert-rpm-package-to-deb-using-alien-package-converter/) tool, uninstall your old dnscrypt and reinstall using the .deb file from alien.

Bookmark this link for post installations: https://packages.altlinux.org/en/sisyphus/srpms/dnscrypt-proxy/

**Reply by pflanzenandi** [reply]
Awesome, thank you! I can feel the speed ;)

**Reply by alpominth** [reply]
You may install `fakeroot` to convert a RPM package to DEB without root privileges.

**Reply by pflanzenandi** [reply]
It worked with sudo for me: `sudo alien --to-deb --scripts dnscrypt.rpm`
